### PR TITLE
Initial changes

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -9,6 +11,49 @@ body {
 	font: 15px Arial, Helvetica, sans-serif;
```
Comment (resumerise) [inline css/style.css]: Use `1.1em`

File: templates/categories/awards.eta [hunk 2]
```diff
@@ -1,23 +1,30 @@
 <section id="awards">
-	<% if(it.resume.settings?.labels?.categoryProperty?.award?.categoryTitle) { %>
+	<div class="item content content__container">
```
Comment (resumerise) [inline templates/categories/awards.eta]: `content__container` is not a child but a specialized element, so therefore `content--container` http://getbem.com/introduction/

File: templates/categories/awards.eta [hunk 3]
```diff
@@ -1,23 +1,30 @@
...
+	<div class="content__container__heading">
```
Comment (resumerise) [inline templates/categories/awards.eta]: `content--container__heading`

File: templates/categories/basics.eta [hunk 4]
```diff
@@ -1,30 +1,41 @@
 <section id='basics'>
-    <%~ include('profile-picture', {
+        <%~ include('profile-picture', {
         picture: it.resume.basics.picture
     }) %>
+    <div class="item content content__container">
```
Comment (resumerise) [inline templates/categories/basics.eta]: See last comment about `content--container`. Please refactor here and everywhere else regarding BEM.

File: templates/layout.eta [hunk 5]
```diff
@@ -14,6 +14,9 @@
 				font-family: 'Poppins', sans-serif;
```
Comment (resumerise) [inline templates/layout.eta]: Add this css to `style.css` and remove it from here.

File: templates/layout.eta [hunk 6]
```diff
@@ -14,6 +14,9 @@
...
 			}
 		</style>
+		<link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@100;300;400;500;600;700&display=swap" rel="stylesheet">
```
Comment (resumerise) [inline templates/layout.eta]: Please import directly into style.css

**Review by ghost — COMMENTED** [review]
Please see comments

File: templates/categories/skills.eta [hunk 7]
```diff
@@ -1,30 +1,33 @@
...
 			<%= it.resume.settings.labels.categoryProperty.skill.categoryTitle %>
 		</h1>
-	<% } %>
+	   <% } %>
 	<% it.resume?.skills.forEach(function(skill) { %>
-		<div class="item content">
-			<%~ include('nav', {categoryKey: it.resume?.settings?.categoryKeys?.skill}) %>
-			<%~ include('key-value-item', {
-				label: it.resume.settings?.labels?.categoryProperty?.skill?.name, 
-				value: skill.name,
```
Comment (resumerise) [inline templates/categories/skills.eta:12]: Why it has been deleted ?!

File: templates/categories/projects.eta [hunk 8]
```diff
@@ -1,15 +1,28 @@
...
+	        <% it.resume?.projects.forEach(function(project) { %>
+
 			<%~ include('nav', {categoryKey: it.resume?.settings?.categoryKeys?.project, showDelete: true, id: project.id}) %>
 			<%~ include('key-value-item', {
 				label: it.resume.settings?.labels?.categoryProperty?.project?.title, 
 				value: project.title,
 				showLabel: false,
 				id: 'project-title'
+			}) %>
+				<%~ include('key-value-item', {
```
Comment (resumerise) [inline templates/categories/projects.eta]: No reason to add it twice, it's a duplicate

**Pull request merged**